### PR TITLE
[RedSuns Review]Add YOLOX(PyTorch) Tutorial

### PR DESCRIPTION
**Pull request by ueno-hiroshi002**
## Pull Request Description:
* Add YOLOX(PyTorch) Tutorial
* Add guide to task_notebooks in README.md of root of tutorial and README.md of task_notebooks.

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [x] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb [hunk 1]
```diff
@@ -0,0 +1,750 @@
...
+   "id": "7c766698",
+   "metadata": {},
+   "source": [
+    "### Various Settings\n",
+    "Here, you can configure the parameters listed below.  \n",
+    "\n",
+    "#### Parameter setting\n",
+    "- IMG_HEIGHT, IMG_WIDTH  \n",
+    "  This parameter allows you to set the size of input images.\n",
+    "- SCORE_THR  \n",
```
Comment (yt0705) [inline tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb:166]: > SCORE_THR  
>     "  This parameter allows you to set the threshold of class score for evaluation.\n",

This parameter is used not only for evaluation but also as the threshold for NMS.

File: tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb [hunk 2]
```diff
@@ -0,0 +1,750 @@
...
+   "cell_type": "markdown",
+   "id": "436710a6",
+   "metadata": {},
+   "source": [
+    "Next, we add the CustomLayer (edgemdt_cl) as post-processing.  \n",
+    "\n",
+    "* FasterRCNNBoxDecode: Decodes YOLOX inference results from Anchor format to BoundingBox format.    \n",
+    "* MulticlassNMS: Executes the Non-Maximum Suppression to remove overlapping boxes.\n",
+    "\n",
+    "Note: YOLOX returns the data of (xc, yc, h, w) but CustomLayers use the data of (yc, xc, h, w), so convert data to (yc, xc, h, w) before decoding."
```
Comment (yt0705) [inline tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb]: >Note: YOLOX returns the data of (xc, yc, h, w) but CustomLayers use the data of (yc, xc, h, w), so convert data to (yc, xc, h, w) before decoding.

Please change the phrasing to “constraints of FasterRCNNBoxDecode” instead of “constraints of CustomLayers”.

File: tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb [hunk 3]
```diff
@@ -0,0 +1,750 @@
...
+    "             score_threshold: float = 0.1):\n",
+    "    \"\"\"\n",
+    "    Evaluation of the COCO dataset.\n",
+    "\n",
+    "    Args:\n",
+    "        model (torch.nn.Module): Evaluation model.\n",
+    "        val_dataloader (DataLoader): Evaluation dataset.\n",
+    "        score_threshold (float): Score threshold.\n",
+    "    \"\"\"\n",
+    "    device = torch.device(\"cuda:0\" if torch.cuda.is_available() else \"cpu\")\n",
```
Comment (yt0705) [inline tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb:634]: > device = torch.device(\"cuda:0\" if torch.cuda.is_available() else \"cpu\")\n",

Please pin it to the CPU to match the YOLOXPostProcess class.

Alternatively, if you need to use CUDA, you will need to modify the YOLOXPostProcess or torch.load sections.

**Review by yt0705 — CHANGES_REQUESTED** [review]

File: tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb (hunk 1 above)
Comment (ueno-hiroshi002) [reply tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb:166]: fixed at b645cfd

File: tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb (hunk 2 above)
Comment (ueno-hiroshi002) [reply tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb]: fixed at b645cfd

File: tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb (hunk 3 above)
Comment (ueno-hiroshi002) [reply tutorials/notebooks/task_notebooks/pytorch/example_yolox_pytorch_mixed_precision_ptq.ipynb:634]: modifed the YOLOXPostProcess or torch.load sections at b645cfd.

**Review by yt0705 — APPROVED** [review]
